### ISSUE TITLE
Does the global_step go back to 0 every time we train a new model? 

### ISSUE THREAD

**Issue by weedwind** [tensorflow/minigo#98]
In the code, we generate a model after ~1000 batches,  then we use the new model to generate games, and train another model. The question is every time when we train a new model, is the global_step in  train_ops reset to 0? If that is the case, how can the learning rate reach the boundary (at 1e6 and 2e6) ?

**Comment by amj** [comment]
It does not reset, it is saved with the rest of the weights at the end of training.
